docs/recipes/nunjucks.md

```diff
@@ -72,7 +72,17 @@ gulp.task('views', () => {
 
 This compiles `app/*.njk` files into static `.html` files in the `.tmp` directory.
 
-### 5. Add `views` as a dependency of both `html` and `serve`
+### 5. Create a 'views_reload' task
```

Comment:
I would prefer that we use `:` for separating words instead of `_`. Could you change this to `views:reload`?